Tests/smokeTest.js

```diff
@@ -0,0 +1,28 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
+describe ('001: Smoke test', function(){
+
+    browser.waitForAngularEnabled(false);
+    browser.ignoreSynchronization = true;
+    browser.manage().window().maximize();
+    var EC = protractor.ExpectedConditions;
+
+    it ("001: Test Case", async function (){
```

Comment:
Add test case title and test suite title

Reply:
I changed both titles